src/main/java/org/isf/lab/gui/elements/PatientComboBox.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2020 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.isf.lab.gui.elements;
+
+import java.util.List;
+import java.util.Optional;
+
+import javax.swing.JComboBox;
+
+import org.isf.generaldata.MessageBundle;
+import org.isf.lab.model.Laboratory;
+import org.isf.patient.model.Patient;
+
+public class PatientComboBox extends JComboBox {
```

Comment:
is this new element meant to be specifically in Lab module rather than in utils/jobject or other upper/common package?

Reply:
It is currently only used in the lab module.  If other uses are found then it can be moved to a more common location.

Reply:
sound fine, thanks!